<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PackageItemForm</class>
 <widget class="QWidget" name="PackageItemForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>266</width>
    <height>251</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="nameLabel">
       <property name="text">
        <string>Name</string>
       </property>
       <property name="buddy">
        <cstring>nameValue</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="nameValue"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="displayLabel">
       <property name="text">
        <string>Display</string>
       </property>
       <property name="buddy">
        <cstring>displayValue</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="displayValue"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="authorLabel">
       <property name="text">
        <string>Author</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="authorValue"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="licenseLabel">
       <property name="text">
        <string>License</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="licenseValue"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="entriesLabel">
       <property name="text">
        <string>Entities</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLabel" name="entriesValue">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="architecturesLabel">
       <property name="text">
        <string>Architectures</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLabel" name="architecturesValue">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Close|QDialogButtonBox::Discard</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
